blazor/common/data-binding/bind-entity-framework.md

```diff
@@ -59,6 +65,236 @@ Now, select Target Framework as (.NET 6.0 or .NET 7.0) in the project template a
 
 ## Creating DbContext and model class
 
+### DbContext and model class in Blazor Web App
+
+Now, scaffold DbContext and model classes from the existing Library database. To perform scaffolding and work with the SQL Server database in our application, you need to install the following NuGet packages. If you have created a Blazor Web App with the `Interactive render mode` set to `WebAssembly` or `Auto`, ensure to follow these steps:
```

Comment:
please avoid to define words in pascal casing unnecessarily. If it is technical word then it is fine other wise use camel casing throughout the sentence.

Reply:
Above mentioned standards updated in existing document.